docs/OV_Runtime_UG/supported_plugins/Device_Plugins.md

```diff
@@ -10,15 +10,14 @@
    openvino_docs_OV_UG_supported_plugins_CPU
    openvino_docs_OV_UG_supported_plugins_GPU
    openvino_docs_OV_UG_supported_plugins_GNA
-   openvino_docs_OV_UG_supported_plugins_ARM_CPU
```

Comment:
keep?

Reply:
This line is not needed since it says that there is a dedicated child page related to ARM plugin.
ARM device link is presented on this page (line 20) and follows to CPU page